examples/platform/silabs/BaseApplication.cpp

```diff
@@ -24,20 +24,21 @@
 #include "AppConfig.h"
 #include "AppEvent.h"
 #include "AppTask.h"
-
 #include <app/server/Server.h>
 
+#define APP_ACTION_BUTTON 1
```

Comment:
We should move this to the platformAbstraction to avoid re-defining each time the GetButtonState is necessary.

Reply:
ok will add const in the SilabsPlatform class in a subsequent pr